### ISSUE TITLE
There was a problem exporting objects and scenes

### ISSUE THREAD

**Issue by zhanghaidi** [mrdoob/three.js#25675]
### Description

After uploading an fbx model, an error occurred when trying to export objects and scenes:

Because my model is relatively large and has many levels, I suspect it is the problem of length exceeding

<img width="1910" alt="1678848698947" src="https://user-images.githubusercontent.com/26913173/225193060-93ca998e-6f2f-442d-87a0-0578eb90320a.png">

### Reproduction steps

1.
2.
3.

### Code

```js

Uncaught RangeError: Invalid string length
    at JSON.stringify (<anonymous>)
    at UIRow.<anonymous> (Menubar.File.js:170:18)

```
![1678849193777](https://user-images.githubusercontent.com/26913173/225194030-8eef8a96-bb4d-4ab7-a2a6-0b299cdbb680.jpg)

### Live example

* [jsfiddle-latest-release](https://jsfiddle.net/g3atw6k5/)
* [jsfiddle-dev](https://jsfiddle.net/hjqw94c5/)

### Version

150

### Browser

Chrome, Edge

### OS

Windows, Linux

**Comment by Mugen87** [comment]
Do you mind sharing the FBX asset in this thread?

**Comment by zhanghaidi** [comment]
Sorry, I can't upload files larger than 25M because my fbx file size is around 80M,

**Comment by zhanghaidi** [comment]
> Do you mind sharing the FBX asset in this thread?

Sorry, I can't upload files larger than 25M because my fbx file size is around 80M, My model has a total of more than 3000 pieces, and I tried exporting them in batches without any problems. If I export them together, it will exceed the memory limit of JSON.stringify

**Comment by Mugen87** [comment]
Looking at the error message again, it seems this issue is a duplicate and can be merged into #9187.

The current approach of the editor for saving/restoring has problems with very large assets. I suggest you try to use the GLTF exporter outside of the editor like demonstrated in [misc_exporter_gltf](https://threejs.org/examples/misc_exporter_gltf) to avoid this issue.
